src/platform/forms-system/src/js/utilities/file/ShowPdfPassword.jsx

```diff
@@ -83,7 +83,7 @@ const PasswordLabel = () => (
 
 const PasswordSuccess = () => (
   <>
-    <p>PDF password</p>
+    <p className="vads-u-margin-top--2">PDF password</p>
```

Comment:
I saw on the screenshot this gets masked, but does this need the explicit classname for that?

Reply:
This class adds padding between the sections. There is CSS that removes all `<p>` tag padding for some reason. Datadog is set up to hide all the input values, so we don't need to do anything here.